routes/orders.js

```diff
@@ -3,12 +3,20 @@ import express from 'express'
 
 const router = express.Router()
 
-router.get('/', (req, res) => {
-  models.Orders.findAll()
-  .then((orders) => {
-    console.log(orders)
+router.get('/', async (req, res) => {
+  let orders
+  try {
+    orders = await models.orders.findAll()
+  } catch (err) {
+    console.log(err)
+    res.sendStatus(500)
+    return orders
```

Comment:
This will be equivalent to returning `null`, since `orders` is not defined. For the sake of being concise, can we just do `return`?